src/modules/logs/logs.actions.ts

```diff
@@ -1,12 +1,38 @@
 import { Action } from '@ngrx/store';
 import { Log } from '../../shared/models/log.model';
 
-export const SAVE_LOG = 'Save Log';
+export const SAVE_LOG = '[Anywhere] Save Log';
```

Comment:
How about [GLOBAL] or [SHARED]? I don't really like Anywhere

Reply:
We can agree on *[GLOBAL]* for these kind of actions, I like it.